apps/chat/src/components/Chat/Chat.tsx

```diff
@@ -725,35 +741,50 @@ export const ChatView = memo(() => {
                               Conversation,
                               Message,
                               number,
-                            ]) => (
-                              <div
-                                key={conv.id}
-                                className={`${
-                                  isCompareMode &&
-                                  selectedConversations.length > 1
-                                    ? 'w-[50%]'
-                                    : 'w-full'
-                                }`}
-                              >
-                                <div className="size-full">
-                                  <MemoizedChatMessage
-                                    key={conv.id}
-                                    message={message}
-                                    messageIndex={index}
-                                    conversation={conv}
-                                    isLikesEnabled={enabledFeatures.has(
-                                      Feature.Likes,
-                                    )}
-                                    editDisabled={!!notAllowedType}
-                                    onEdit={onEditMessage}
-                                    onLike={onLikeHandler(index, conv)}
-                                    onDelete={() => {
-                                      handleDeleteMessage(index);
-                                    }}
-                                  />
+                            ]) => {
+                              if (
```

Comment:
I think it's bad practise to set any usual logic inside the rendered HTML like changing state